test/parallel/test-repl.js

```diff
@@ -142,7 +142,7 @@ function error_test() {
       expect: prompt_unix },
     // But passing the same string to eval() should throw
     { client: client_unix, send: 'eval("function test_func() {")',
-      expect: /\bSyntaxError: Unexpected end of input/ },
+      expect: /^\bSyntaxError: Unexpected end of input/ },
```

Comment:
1. There are some RegExps with the same pattern which are left unchanged (lines 193, 198, 203 etc).

2. It seems the `\b` symbol after the `^` is unneeded.